mesonbuild/backend/vs2010backend.py

```diff
@@ -413,8 +413,8 @@ def gen_run_target_vcxproj(self, target, ofname, guid):
         cmd = [sys.executable, os.path.join(self.environment.get_script_dir(), 'commandrunner.py'),
                self.environment.get_build_dir(),
                self.environment.get_source_dir(),
-               self.get_target_dir(target),
-               get_meson_script(self.environment, 'mesonintrospect')]
+               self.get_target_dir(target)] + \
+               self.environment.get_build_command() + ['introspect']
```

Comment:
__[flake8]__
* [E127] continuation line over-indented for visual indent

<!-- Comment by [SideCI](https://sideci.com) -->